tests/services/test_serializers.py

```diff
@@ -1,91 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
removed because:
1- unused... all skiped
2- serializers are now at resource level, this will be added when extensions are revisited?